src/org/parosproxy/paros/extension/option/OptionsConnectionPanel.java

```diff
@@ -854,7 +858,18 @@ private JCheckBox getCheckBoxSingleCookeRequestHeader() {
         return checkBoxSingleCookieRequestHeader;
     }
 
-    private ProxyExcludedDomainsMultipleOptionsPanel getProxyExcludedDomainsPanel() {
+    public JCheckBox getCheckBoxHttpStateEnabled() {
+    	if (checkBoxHttpStateEnabled == null) {
+    		checkBoxHttpStateEnabled = new JCheckBox(Constant.messages.getString("conn.options.httpStateEnabled"));
+    	}
+		return checkBoxHttpStateEnabled;
+	}

```

Comment:
Can't wait for https://github.com/zaproxy/zap-api-java/pull/32 to come in place ...